Documentation/policy.rst

```diff
@@ -188,6 +188,34 @@ the label ``env=prod`` in order for access to be granted::
 
 .. _policy_cidr:
 
+Layer 3: Entities
+~~~~~~~~~~~~~~~~~
+
+There is an additional syntactic sugar for explicitly whitelisting two labeled entities: ``world`` and ``host``
+Instead of using verbose ``fromEndpoints`` structure::
+
+        [{
```

Comment:
I think we can leave this part out and just provide an example on how to use it. Ideally, the user will never see the reserved labels again.

Reply:
Done